pkg/formatted/color.go

```diff
@@ -16,19 +16,78 @@ package formatted
 
 import (
 	"io"
+	"sync"
+	"sync/atomic"
 
 	"github.com/fatih/color"
 )
 
+var (
+	// Red is avoided as keeping it for errors
+	palette = []color.Attribute{
+		color.FgHiGreen,
+		color.FgHiYellow,
+		color.FgHiBlue,
+		color.FgHiMagenta,
+		color.FgHiCyan,
+	}
+)
+
+type atomicCounter struct {
```

Comment:
I liked your earlier name: counter but I am not opposed to this (internal implementation)